src/meson.build

```diff
@@ -48,8 +45,16 @@ conf_paths.set_quoted('DATADIR', join_paths(get_option('prefix'), get_option('da
 configure_file(output: 'config_paths.hpp',
                configuration : conf_paths)
 
+# Compile stb_image implementations separatelly.
+# This allows to build project with Meson --unity=on option, for faster compilation
+# and potentially better optimization.
+stb_image_lib        = static_library('stb_image',        'stb_image.cpp')
+stb_image_resize_lib = static_library('stb_image_resize', 'stb_image_resize.cpp')
+logger_lib           = static_library('logger',           'logger.cpp')
```

Comment:
What's the problem with `logger.cpp`?

Reply:
I have no idea. It produces errors, so I excluded it too. In theory it should work, but require some additional attention.